benchmarks/src/garage_benchmarks/experiments/algos/ppo_garage_tf.py

```diff
@@ -52,7 +52,7 @@ def ppo_garage_tf(ctxt, env_id, seed):
                 optimizer_args=dict(
                     batch_size=32,
                     max_epochs=10,
-                    tf_optimizer_args=dict(learning_rate=3e-4),
+                    learning_rates=dict(learning_rate=3e-4),
```

Comment:
i apologize for not being clear -- there are optimizer_args other than learning rate.

What i meant was that we would not have the dict at all, i.e. don't support passing arbitrary kwargs down to the tf.Optimizer instance.

e.g.

```
optimizer_args=dict(
    batch_size=32,
    max_epochs=10,
    learning_rate=3e-4,
)
```